test/e2e/data/opt_125m_input_generate.json

```diff
@@ -11,5 +11,6 @@
       }
     ],
     "temperature": 0,
-    "max_tokens": 20
+    "max_tokens": 20,
+    "chat_template": "{% for message in messages %}{{ message.content }}{{ eos_token }}{% endfor %}"
```

Comment:
The test should still work without adding chat_template field right?

Reply:
@yuzisun for this model, its tokenizer does not have a default chat_template. The default chat_template was removed in src/transformers/models/gpt2/tokenization_gpt2.py in https://github.com/huggingface/transformers/pull/31733. Thus, you either assign a chat_template to the tokenizer as in tests/models/gpt2/test_tokenization_gpt2.py in https://github.com/huggingface/transformers/pull/31733, or give an explicit chat_template parameter as tests/async_engine/test_chat_template.py in https://github.com/vllm-project/vllm/pull/7238 (also this pr). Otherwise it raise error : https://github.com/huggingface/transformers/blob/37ea04013b34b39c01b51aeaacd8d56f2c62a7eb/src/transformers/tokenization_utils_base.py#L1964C5-L1964C34 and this test would not pass.